code/config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "root",
```

Comment:
This will cause a problem... we might have to change this back to `postgres` on each of our machines